src/server/models/room/ParticipantRoom.ts

```diff
@@ -112,12 +112,16 @@ export class ParticipantRoom implements Room {
             throw new Error("Runtime error: Override issue - not receiving events in a HubRoom");
         }
 
+        // Check the event is valid.
+        // @ts-ignore
```

Comment:
hmm, actually - what is this `tsignore` doing?

Reply:
Oh, because it requires a PDU and will break with an LPDU, but I think this should always be a PDU at this point?

Reply:
we should be casting rather than ignoring the error :p

Reply:
Will update it. I want to double check the code paths of how we get here.

Reply:
The callers always pass in a `PDU` as far as I can tell.